src/components/ImageSearch.vue

```diff
@@ -103,6 +174,10 @@
                     this.flickrImageItems = [];
                     this.query();
                 }
+                if(this.bucket_count > 1)
+                {
+                  this.multiple_buckets = this.bucket_count;
```

Comment:
What happens here, if I switch back to a keyword, that has only one bucket? -> That would cause an error, because mulitple buckets count would be still the old one, right?